hms-infrastructure/cdk.context.json

```diff
@@ -0,0 +1,6 @@
+{
+  "hosted-zone:account=329504430700:domainName=hackweek.link:region=eu-central-1": {
+    "Id": "/hostedzone/Z0913911SWOKY9EUXMON",
+    "Name": "hackweek.link."
```

Comment:
should this link be part of the public repo?

Reply:
no